pkg/kubelet/managed/managed.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package managed
+
+import (
+	"fmt"
+	"os"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/klog/v2"
+)
+
+var (
+	pinnedManagementEnabled      bool
+	pinnedManagementFilename     = "/etc/kubernetes/openshift-workload-pinning"
+	WorkloadsAnnotation          = "workload.openshift.io/management"
+	ManagedCapacityLabel         = "management.workload.openshift.io/cores"
```

Comment:
Eventually we will want these names to be built based on workload type name in the incoming annotation, so we can support multiple workload types. Do we want to go ahead and future-proof the implementation now? Or leave that for later?

Reply:
updated to support other workload types.